panther_manager/include/panther_manager/manager_bt_node.hpp

```diff
@@ -45,11 +60,20 @@ class ManagerNode
   BT::Tree safety_tree_;
   BT::Tree shutdown_tree_;
 
-  void setup_behavior_tree(BT::Tree & tree, BT::NodeConfig & config, const std::string tree_name);
+  MovingAverage<double> battery_temp_ma_;
+  MovingAverage<double> cpu_temp_ma_;
+  MovingAverage<double> front_driver_temp_ma_;
+  MovingAverage<double> rear_driver_temp_ma_;
+
+  void battery_cb(const sensor_msgs::BatteryState::ConstPtr & battery);
+  void driver_state_cb(const panther_msgs::DriverState::ConstPtr & driver_state);
   void e_stop_cb(const std_msgs::Bool::ConstPtr & e_stop);
   void io_state_cb(const panther_msgs::IOState::ConstPtr & io_state);
+  void system_status_cb(const panther_msgs::SystemStatus::ConstPtr & system_status);
   void safety_tree_timer_cb();
   void lights_tree_timer_cb();
+  void shutdown_robot(const std::string & message);
+  BT::NodeConfig create_bt_config(const std::map<std::string, std::any> bb_values = {});
```

Comment:
```suggestion
  BT::NodeConfig create_bt_config(const std::map<std::string, std::any> bb_values = {}) const;
```

This function is not modifying anything within class, hence can be `const`